sigle/src/modules/layout/components/AppHeader.tsx

```diff
@@ -4,14 +4,26 @@ import {
   TwitterLogoIcon,
   DiscordLogoIcon,
   SunIcon,
+  HamburgerMenuIcon,
```

Comment:
regarding the mobile menu of this one, will it be impacted by the coming changes that we will have with the top right corner? It seems to me that these changes should be made in a separate pull request.